themes/ezcontent_theme/ezcontent_theme.libraries.yml

```diff
@@ -3,24 +3,23 @@ framework:
     theme:
       css/glyphicons.css: {}
       css/style.css: {}
-      css/paragraph_type_map.css: {}
-      css/paragraph_type_social.css: {}
-      css/paragraph_type_embed.css: {}
-      css/paragraph_type_hero_media.css: {}
-      css/paragraph_type_card.css: {}
-      css/paragraph_type_card_list.css: {}
-      css/paragraph_type_asset.css: {}
-      css/paragraph_type_gallery.css: {}
-      css/paragraph_type_gallery_carousel.css: {}
-      css/paragraph-type-quotes.css: {}
-      css/paragraph_type_faq.css: {}
-      css/paragraph_type_form.css: {}
-      css/paragraph_type_referenced.css: {}
-      css/paragraph_type_view_list.css: {}
-      css/block_type_hero_media.css: {}
-      css/block_type_map.css: {}
-      css/block_type_embed.css: {}
-      css/block_type_socialmedia.css: {}
+      css/paragraph/paragraph_type_map.css: {}
+      css/paragraph/paragraph_type_social.css: {}
+      css/paragraph/paragraph_type_embed.css: {}
+      css/paragraph/paragraph_type_hero_media.css: {}
+      css/paragraph/paragraph_type_card.css: {}
+      css/paragraph/paragraph_type_card_list.css: {}
+      css/paragraph/paragraph_type_asset.css: {}
+      css/paragraph/paragraph_type_gallery.css: {}
+      css/paragraph/paragraph_type_gallery_carousel.css: {}
+      css/paragraph/paragraph-type-quotes.css: {}
+      css/paragraph/paragraph_type_faq.css: {}
+      css/paragraph/paragraph_type_form.css: {}
+      css/paragraph/paragraph_type_view_list.css: {}
+      css/block/block_type_hero_media.css: {}
+      css/block/block_type_map.css: {}
+      css/block/block_type_embed.css: {}
+      css/block/block_type_socialmedia.css: {}
```

Comment:
@sumit3021kumar Since we are placing these files into directories now, shall we rename them to match the component ? like `paragraph_type_map.css` can be renamed as `map.css`

Reply:
Hey, @vaibhavjain-in changes are done for both paragraphs and block. Please, can you review it again?